db/index.js

```diff
@@ -1,12 +1,69 @@
-var db = mysql.createConnection({
-	host     : '127.0.0.1',
-	user     : 'root',
-	database : 'zobinHood'
+const mysql = require('mysql');
+const seedData = require('./seedData.js');
+
+seedData.pricingArr;
+seedData.generator();
+
+const conn = mysql.createConnection({
+  host     : '127.0.0.1',
+  user     : 'root',
+  database : 'zobinHood',
 });
 
-db.connect((err) => {
-	if (err) console.log('DB connection error: ' err);
-	else console.log('Connected to zobinHood db!')
+conn.connect((err) => {
+  if (err) {
+    console.log('DB connection error: ', err);
+  } else {
+    console.log('Connected to zobinHood db!');
+  }
 });
 
-module.exports = db;
\ No newline at end of file
+const tableCreateStr = `CREATE TABLE IF NOT EXISTS pricing (
+  priceid INT NOT NULL AUTO_INCREMENT,
+  price INT NOT NULL,
+  Listing_id CHAR(3) NOT NULL,
+  PRIMARY KEY ( priceid )
+)`;
+
+const buildPackage = () => {
+  // db.query('CREATE DATABASE IF NOT EXISTS zobinHood', (err, results) => {
+  //   if (err) console.log(err);
+  //   console.log('db created: ', results);
+  // });
+
+  conn.query('USE zobinHood', (err, results) => {
+    if (err) console.log(err);
+    console.log('db zobinHood in use: ', results);
+  });
+
+  conn.query(tableCreateStr, (err, results) => {
+    if (err) console.log(err);
+    console.log('pricing table created: ', results);
+  });
+
+  conn.query('TRUNCATE TABLE pricing', (err, results) => {
+    if (err) console.log(err);
+    console.log('pricing table ready for new data...', results);
+  });
+
+  const dbSeed = () => {
+    for (let i = 0; i < seedData.pricingArr.length; i++) {
+      let p = seedData.pricingArr[i].price;
+      let l = seedData.pricingArr[i].Listing_id;
+      let queryString = `INSERT INTO pricing (price, Listing_id)
+        VALUES ('${p}', '${l}')`;
```

Comment:
remember to escape the inserts with MySql's built in tools. Refer to Josh's movie list practice that we did, if you forgot. I could be totally wrong too. I'll look into it after I finish reviewing.

Reply:
Great point. I'll take a look back at it, is super important for the 'inputs' (that are the rest of my components) to not be vulnerable to sql injection.